### PR TITLE
chore: use recommended restrict-plus-operands

### PR DESCRIPTION
**Pull request by Shinigami92**
Use the recommended state of the rule
https://typescript-eslint.io/rules/restrict-plus-operands/

### CODE REVIEW

File: src/image.ts [hunk 1]
```diff
@@ -264,22 +264,11 @@ export class Image {
...
-      '" font-size="20" alignment-baseline="middle" text-anchor="middle" fill="white">' +
-      width +
-      'x' +
-      height +
-      '</text></svg>';
+    const svgString = `<svg xmlns="http://www.w3.org/2000/svg" version="1.1" baseProfile="full" width="${width}" height="${height}"><rect width="100%" height="100%" fill="${color}"/><text x="${
+      width / 2
+    }" y="${
+      height / 2
+    }" font-size="20" alignment-baseline="middle" text-anchor="middle" fill="white">${width}x${height}</text></svg>`;
```
Comment (ST-DDT) [inline src/image.ts:271]: This isn't really readable. Maybe add use an array for the elements and join it afterwards?

**Review by ST-DDT — APPROVED** [review]

File: src/image.ts (hunk 1 above)
Comment (Shinigami92) [reply src/image.ts:271]: We need to improve readability for such things anyways, we could address this in a dedicated PR.
This was currently converted by automation.

**Review by ST-DDT — APPROVED** [review]

File: src/vehicle.ts [hunk 2]
```diff
@@ -76,16 +76,15 @@ export class Vehicle {
...
-      this.faker.random.alpha({
-        count: 1,
-        upcase: true,
-        bannedChars: bannedChars,
-      }) +
-      this.faker.random.alphaNumeric(1, { bannedChars: bannedChars }) +
-      this.faker.datatype.number({ min: 10000, max: 100000 })
-    ) // return five digit #
+    return `${this.faker.random.alphaNumeric(10, {
+      bannedChars: bannedChars,
```
Comment (griest024) [inline src/vehicle.ts:80]: This can be written just as `bannedChars,`. Same in other places.

**Review by griest024 — CHANGES_REQUESTED** [review]

File: src/vehicle.ts (hunk 2 above)
Comment (Shinigami92) [reply src/vehicle.ts]: Not really what the PR was targeting, but anyways, I "fixed" it

**Review by griest024 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**